soc/infineon/cat1b/cyw20829/CMakeLists.txt

```diff
@@ -30,6 +30,78 @@ math(EXPR flash_addr_offset
   OUTPUT_FORMAT HEXADECIMAL
 )
 set(gen_app_header_args --flash_addr_offset ${flash_addr_offset})
+set(app_signed_enc_path ${ZEPHYR_BINARY_DIR}/${KERNEL_NAME})
+
+if(CONFIG_INFINEON_SECURE_LCS OR CONFIG_BOOTLOADER_MCUBOOT)
+  # Check cysecuretools
+  find_program(CYSECURETOOLS cysecuretools)
```

Comment:
should just use the required flag in find_program() rather then re-inventing the wheel